src/main/scala/mesosphere/marathon/state/Group.scala

```diff
@@ -3,14 +3,15 @@ package state
 
 import java.util.Objects
 
+import com.typesafe.scalalogging.StrictLogging
 import com.wix.accord._
 import com.wix.accord.dsl._
 import mesosphere.marathon.api.v2.Validation._
 import mesosphere.marathon.api.v2.validation.AppValidation
 import mesosphere.marathon.core.pod.PodDefinition
 import mesosphere.marathon.plugin.{ Group => IGroup }
-import mesosphere.marathon.state.Group._
-import mesosphere.marathon.state.PathId._
+import mesosphere.marathon.state.Group.{ defaultApps, defaultPods, defaultGroups, defaultDependencies, defaultVersion }
+import mesosphere.marathon.state.PathId.{ validPathWithBase, StringPathId }
```

Comment:
none of these imports seem to be used... are they implicit?  if they are can we call them out?

Reply:
`validPathWithBase` is used in line 137. `StringPathId` is indeed an explicit import of an implicit. It's used in line 195 for `group.id.map(_.toPath)`.